src/adapters/MeetingsJSONAdapter.test.js

```diff
@@ -43,6 +43,17 @@ describe('Meetings JSON Adapter', () => {
     );
   });
 
+  test('getMeeting() renders the local media if localVideo property is defined', (done) => {
+    window.navigator.mediaDevices = {
+      getUserMedia: () => jest.fn(Promise.resolve()),
```

Comment:
Would like to see this resolve something, and then checked in the expectation:

```suggestion
      getUserMedia: () => jest.fn(Promise.resolve('mock-stream')),
```